src/libcore/tests/tuple.rs

```diff
@@ -8,18 +8,19 @@ fn test_clone() {
 }
 
 #[test]
-fn test_tuple_cmp() {
+fn test_partial_eq() {
     let (small, big) = ((1, 2, 3), (3, 2, 1));
-
-    let nan = 0.0f64/0.0;
-
-    // PartialEq
     assert_eq!(small, small);
     assert_eq!(big, big);
-    assert!(small != big);
-    assert!(big != small);
+    assert_ne!(small, big);
+    assert_ne!(big, small);
+}
+
+#[test]
+fn test_partial_ord() {
+    let (small, big) = ((1, 2, 3), (3, 2, 1));
+    let nan = 0.0f64/0.0;
```

Comment:
This should be `core::f64::NAN`.

Even better, import it and use it instead of the variable:

```rust
use core::f64::NAN;

// ...

assert!(!((1.0f64, 2.0f64) >= (NAN, 3.0)))
```